packages/core/src/Stores/client-store.js

```diff
@@ -1217,7 +1217,8 @@ export default class ClientStore extends BaseStore {
                 console.error('JSON parse failed, invalid value (client.accounts): ', error);
             }
 
-            const { oauth_token, client_id } = response.new_account_real ?? response.new_account_maltainvest;
+            const { oauth_token, client_id, currency_type } =
+                response.new_account_real ?? response.new_account_maltainvest;
```

Comment:
I believe this should be || instead of ??. 
On first load it could be undefined and always goes to maltainvest. 
Since it can be the either 2 of those calls, its better go with OR operator

Reply:
@thisyahlen-deriv This part I didn't touch the logic but I think the only difference here if I replace it with `||` is we will be checking falsy values as well not only `null` and `undefined` which won't make any difference, even if it is `||` and on first load is undefined it will go maltainvest again I think `||` won't fix that concern you have here , correct me if I'm wrong

Reply:
If you think, this wont be any affect. Then you can leave it as it is.